apps/next-app/src/components/intro/cardsPanel.tsx

```diff
@@ -0,0 +1,36 @@
+// var cards = [
+//   {
+//     icon: 'icon',
+//     title: 'Gia tri cot loi',
+//     description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+//   },
+//   {
+//     icon: 'icon',
+//     title: 'Gia tri cot loi',
+//     description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+//   },
+//   {
+//     icon: 'icon',
+//     title: 'Gia tri cot loi',
+//     description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+//   }
+// ]
+
+const IntroPanelCards = ({cards}: Array<{
+  icon: string;
+  title: string;
+  description: string;
+}>) => {
+  return (
+    <div className="">
+      {cards.map((card: {
+        icon: string;
+        title: string;
+        description: string;
+      }) => {
+        <img src={card.icon}/>
+        // code logic here
+      })}
+    </div>
+  );
+};
```

Comment:
```suggestion
  description: string;
}>) =>(
    <div className="">
      {cards.map((card: {
        icon: string;
        title: string;
        description: string;
      }) => {
        <img src={card.icon}/>
        // code logic here
      })}
    </div>
  );
```
redundant wrapper here maybe?


*small, can resolve in later PR